airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/lib/model.ts

```diff
@@ -0,0 +1,3 @@
+import { ReleaseStage } from "core/request/AirbyteClient";
+
+export const freeReleaseStages: ReleaseStage[] = ["alpha", "beta"];
```

Comment:
Extracting this from the free tag component file is 100% the right move, thanks. Extremely optional suggestion for the API:
```suggestion
export const isFreeReleaseStage = (stage: ReleaseStage): boolean => ["alpha", "beta"].includes(stage);
```